File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/CreateNewActivityInjector.java
package ua.meugen.android.pdfcomposer.ui.activities.createnew;

import ua.meugen.android.pdfcomposer.app.di.AppComponent;
import ua.meugen.android.pdfcomposer.ui.activities.base.Injector;

/**
 * @author meugen
 */
class CreateNewActivityInjector implements Injector {

    private final CreateNewActivity activity;

    CreateNewActivityInjector(final CreateNewActivity activity) {
        this.activity = activity;
    }

    @Override
    public void inject(final AppComponent component) {
        activity.fragmentManager = activity.getSupportFragmentManager();
    }
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/CreateNewActivityInjector.kt
package ua.meugen.android.pdfcomposer.ui.activities.createnew

import ua.meugen.android.pdfcomposer.app.di.AppComponent
import ua.meugen.android.pdfcomposer.ui.activities.base.Injector

/**
 * @author meugen
 */
internal class CreateNewActivityInjector(private val activity: CreateNewActivity) : Injector {

    override fun inject(component: AppComponent) {
        activity.fragmentManager = activity.supportFragmentManager
    }
}
